ibis/expr/scope.py

```diff
@@ -168,14 +212,16 @@ def make_scope(
     ----------
     op: ibis.expr.operations.Node
         key in scope.
-    result : Object
-        concrete data, type could be different for different backends.
     timecontext: Optional[TimeContext]
         time context associate with the result.
+    value : Object
+        concrete data, type could be different for different backends.
 
     Returns
     -------
     Scope
         a new Scope instance with op in it.
     """
-    return Scope({op: ScopeItem(result, timecontext)})
+    scope = Scope()
```

Comment:
wy don't you do this in the Scope({op: ....})

Reply:
Yes I should do that, fixed.